### PR TITLE
Added the Orbiting items 3D components

### LINKED ISSUE #226: Orbiting items (3D)
### Description
Create the component as shown in the video attached below. We already have an [orbiting items component](https://animata.design/docs/list/orbiting-items) as this one, but this is more complex and 3d looking.

### Animation Preview
- Screenshot/Video:

https://github.com/user-attachments/assets/5ba932d2-b669-4190-93db-bd62c661b479

> [!NOTE]
> Please refer to the **Animata contributing guidelines** available [here](https://www.animata.design/docs/contributing) for rules on how to contribute to the project. Let us know in the comments if you’re working on this issue, and feel free to ask any questions!

### Requirements
- [ ] Create a new animated component that matches the provided design.
- [ ] Add example usage of the component in the documentation or storybook.

<details>
  <summary>
    <h3> Guidelines & Best Practices</h3>
  </summary>

- Write reusable and clean code following the project’s style guide.
- The component should be **fully customizable**. The user should be able to modify props such as duration, easing, color, and size where applicable.
- Follow the [Contributing Guidelines](https://www.animata.design/docs/contributing) before submitting your pull request.

</details>

> [!IMPORTANT]  
> To ensure more contributors have the opportunity to participate, we kindly request that:
> - Each contributor submits only **one pull request** related to this issue.
> - If you're interested in working on this issue, first check if it has already been claimed. If not, **assign yourself** to the issue and move it to **In Progress** on the project board.

### Additional resources
Posted on X by: https://x.com/oguzyagizkara/status/1699745310312960219

---
### Checklist
- [ ] I have read and followed the contributing guidelines.
- [ ] I have developed the component according to the requirements.
- [ ] I have tested the component across different browsers and screen sizes.
- [ ] I have documented the component for future reference.
- [ ] I have submitted a pull request with appropriate labels and referenced this issue number.

### PR DESCRIPTION
**Pull request by Pikachu-345**
Closes #226 

- Added the Orbiting Items 3D in the "list" section of navbar 
- Updated the docs file for the same component 

Please review my request..and suggest any changes needed..ready to improvise if necessary..

### CODE REVIEW

File: content/docs/list/orbiting-items-3-d.mdx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+<ComponentPreview name="list-orbiting-items-3-d--docs" />
+
+## Installation
+
+<Steps>
+<Step>Install dependencies</Step>
+
+```bash
+npm install framer-motion lucide-react
```
Comment (hari) [inline content/docs/list/orbiting-items-3-d.mdx:15]: your component doesn't need `framer-motion`

File: content/docs/list/orbiting-items-3-d.mdx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+```json
+module.exports = {
+  theme: {
+    extend: {
+    }
+  }
+}
+```
+
```
Comment (hari) [inline content/docs/list/orbiting-items-3-d.mdx:30]: you don't need this step

File: animata/list/orbiting-items-3-d.tsx [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+    const xTilted = x * Math.cos(tiltRadians) - y * Math.sin(tiltRadians);
+    const yTilted = x * Math.sin(tiltRadians) + y * Math.cos(tiltRadians);
+    const zIndex = angle > 180 ? -1 : 1;
+
+    return {
+      left: `${50 + xTilted}%`,
+      top: `${50 + yTilted}%`,
+      transform: "translate(-50%, -50%)",
+      zIndex: zIndex,
+    };
```
Comment (hari) [inline animata/list/orbiting-items-3-d.tsx:121]: we need the scale effect as well, the items closer should scale up and farther should scale down.

**Review by hari — CHANGES_REQUESTED** [review]

**Comment by hari** [comment]
Triggering workflow by closing and opening the PR.

**Comment by hari** [comment]
Build is failing as well:

```

./animata/list/orbiting-items-3-d.stories.tsx:18:3
Type error: Type '{}' is not assignable to type 'Partial<{ radiusX: number; radiusY: number; tiltAngle: number; duration: number; items: ReactNode[]; pauseOnHover?: boolean | undefined; backgroundClassName?: string | undefined; containerClassName?: string | undefined; className?: string | undefined; }> & { ...; }'.
  Type '{}' is missing the following properties from type '{ radiusX: number; radiusY: number; tiltAngle: number; duration: number; items: ReactNode[]; pauseOnHover?: boolean | undefined; backgroundClassName?: string | undefined; containerClassName?: string | undefined; className?: string | undefined; }': radiusX, radiusY, tiltAngle, duration, items

  16 |
  17 | export const Primary: Story = {
> 18 |   args: {},
     |   ^
  19 | };
  ```

File: animata/list/orbiting-items-3-d.tsx (hunk 3 above)
Comment (Pikachu-345) [reply animata/list/orbiting-items-3-d.tsx]: Done!!

File: content/docs/list/orbiting-items-3-d.mdx (hunk 2 above)
Comment (Pikachu-345) [reply content/docs/list/orbiting-items-3-d.mdx]: Done!!

File: content/docs/list/orbiting-items-3-d.mdx (hunk 1 above)
Comment (Pikachu-345) [reply content/docs/list/orbiting-items-3-d.mdx]: Done!!

**Comment by Pikachu-345** [comment]
@hari  made all the required changes please review

**Comment by hari** [comment]
@Pikachu-345 It's looking really good. 

Small changes:
1. Could you use Animata's logo (SVG) in the center and make it animate slowly?
2. Could you play with the background color a little bit to make it look better? Also, add subtle gradient and drop-shadow on the items to make it look like the video.

**Review by hari — APPROVED** [review]

**Comment by Pikachu-345** [comment]
okay will do

**Comment by Pikachu-345** [comment]
@hari done!!

**Review by hari — APPROVED** [review]

**Pull request merged**